Fix for issue #398

dmitryrogozhny opened the pull request:
| Q               | A
| --------------- | ---
| Bug fix?        | [x]
| New feature?    | [ ]
| New sample?      | [ ]
| Related issues?  | fixes #398

#### What's in this Pull Request?

Issue: the Placeholder control is using global class names from Office UI Fabric. When no `fabric.css` file is available (I believe, this is the current default behavior in SharePoint Online), these classes do not work. Icon and text sizes will be smaller than expected in this case.

This PR changes global CSS class names with corresponding mixins, thus the Placeholder control does not depend on Office UI Fabric in runtime.

msftclas commented:
[![CLA assistant check](https://cla.opensource.microsoft.com/pull/badge/not_signed)](https://cla.opensource.microsoft.com/SharePoint/sp-dev-fx-controls-react?pullRequest=399) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.microsoft.com/SharePoint/sp-dev-fx-controls-react?pullRequest=399) before we can accept your contribution.<br/><br/>:x: dmitryrogozhny [sign now](https://cla.opensource.microsoft.com/SharePoint/sp-dev-fx-controls-react?pullRequest=399)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.microsoft.com/check/SharePoint/sp-dev-fx-controls-react?pullRequest=399) it.</sub>

dmitryrogozhny commented:
Modified failing tests for the Placeholder:
- changed selector for the Icon component as the i tag won't have `ms-Icon--Add` class (data-icon-name attribute is set instead).
- renamed duplicate test 'Test placeholder without button' to 'Test that placeholder renders' as the test only checks that component renders.
- disabled a warning message for the Icon component - WARN: 'The icon "Add" was used but not registered. See http://aka.ms/fabric-icon-usage for more information.'
- changed test condition for 'Test placeholder with custom classname' - it should have a top-level <div> container with both standard and custom classes applied.

AJIXuMuK commented:
Thank you @dmitryrogozhny for the fix!
